Sources/BraintreePayPalNativeCheckout/BTPayPalNativeCheckoutClient.swift

```diff
@@ -1,19 +1,88 @@
-import Foundation
 #if canImport(BraintreeCore)
 import BraintreeCore
 #endif
 
-@objcMembers
-public class BTPayPalNativeCheckoutClient: NSObject {
-    private var apiClient: BTAPIClient
-    
+#if canImport(BraintreePayPal)
+import BraintreePayPal
+#endif
+
+import PayPalCheckout
+
+/// Client used to collect PayPal payment methods. If possible, this client will present a native flow; otherwise, it will fall back to a web flow.
+@objc public class BTPayPalNativeCheckoutClient: NSObject {
```

Comment:
It doesn't look like we have much test coverage for this class. Are we able to add some tests for the success/failure paths here?

Reply:
Added tests. the remaining path is where the SDK interacts directly with NXO - so those are still untested from within BT